runtime/runtime_storage.go

```diff
@@ -313,31 +291,6 @@ func (s *runtimeStorage) writeCached(inter *interpreter.Interpreter) {
 
 	// Write cache entries in order
```

Comment:
is this comment still relevant?

Reply:
The comment applied to both parts below, writing the items in order using the high-level storage API, and then again using the low-level API, so yes, it's still relevant